packages/util-dns/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "@aws-sdk/util-dns",
+  "version": "3.0.0",
+  "description": "Implementations of DNS host resolvers.",
+  "main": "./dist-cjs/index.js",
+  "module": "./dist-es/index.js",
+  "scripts": {
+    "build": "concurrently 'yarn:build:cjs' 'yarn:build:es' 'yarn:build:types'",
+    "build:cjs": "tsc -p tsconfig.cjs.json",
+    "build:es": "tsc -p tsconfig.es.json",
+    "build:include:deps": "lerna run --scope $npm_package_name --include-dependencies build",
+    "build:types": "tsc -p tsconfig.types.json",
+    "build:types:downlevel": "downlevel-dts dist-types dist-types/ts3.4",
+    "clean": "rimraf ./dist-* && rimraf *.tsbuildinfo",
+    "test": "jest"
+  },
+  "keywords": [
+    "dns"
+  ],
+  "author": {
+    "name": "AWS SDK for JavaScript Team",
+    "url": "https://aws.amazon.com/javascript/"
+  },
+  "license": "Apache-2.0",
+  "dependencies": {
+    "tslib": "^2.3.1"
+  },
+  "devDependencies": {
+    "@aws-sdk/types": "*",
+    "@tsconfig/recommended": "1.0.1",
+    "@types/node": "^14.14.31",
+    "concurrently": "7.0.0",
+    "downlevel-dts": "0.10.1",
+    "rimraf": "3.0.2",
+    "typedoc": "0.19.2",
+    "typescript": "~4.6.2"
+  },
+  "types": "./dist-types/index.d.ts",
+  "engines": {
+    "node": ">= 14.0.0"
+  },
+  "typesVersions": {
+    "<4.0": {
+      "types/*": [
+        "types/ts3.4/*"
+      ]
+    }
+  },
+  "files": [
+    "dist-*"
+  ],
+  "homepage": "https://github.com/aws/aws-sdk-js-v3/tree/master/packages/util-dns",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/aws/aws-sdk-js-v3.git",
+    "directory": "packages/util-dns"
+  },
+  "browser": {
+    "./dist-es/index": "./dist-es/index.browser",
+    "./dist-es/HostResolver": "./dist-es/HostResolver.browser",
+    "./dist-es/NodeDnsLookupHostResolver": false,
+    "./dist-es/NodeDnsResolveHostResolver": false
+  },
+  "react-native": {
+    "./dist-es/index": "./dist-es/index.browser",
+    "./dist-es/HostResolver": "./dist-es/HostResolver.browser",
+    "./dist-es/NodeDnsLookupHostResolver": false,
+    "./dist-es/NodeDnsResolveHostResolver": false
+  }
```

Comment:
This section needs review since I'm unfamiliar with how these entries work for exports.

Reply:
looks right to me